src/preprocess.c

```diff
@@ -843,7 +843,7 @@ int preprocess(char *source, FILE *f_target, struct constants *constants, struct
         // if (constants[1].value == 0)
         //     constants[1].value = (char *)safe_malloc(16);
         // sprintf(constants[1].value, "%i", line - 1);
-
+        
```

Comment:
Excessive spacing.

Reply:
Backporting fixes via web editor give it a break :)

Reply:
I don't know when you are finished, no "WIP" marker, just saying what flummi would say anyways. :P